api/include/opentelemetry/metrics/meter.h

```diff
@@ -90,7 +90,7 @@ class Meter
    * @param description a brief description of what the Observable Gauge is used for.
    * @param unit the unit of metric values following https://unitsofmeasure.org/ucum.html.
    */
-  virtual nostd::shared_ptr<ObservableInstrument> CreateLongObservableGauge(
+  virtual nostd::shared_ptr<ObservableInstrument> CreateUInt64ObservableGauge(
```

Comment:
It's not explicit in [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#asynchronous-gauge-operations) that the recorded gauge values should be only positive. Which means it can be both positive and negative.  Also, as an example - recorded temperature can be negative.
Sorry for creating confusion, I should have seen it earlier. I think we should change it to `CreateInt64ObservableGauge`

Reply:
thanks, fixed